File: DB_lez06_task_edicola/DB_lez06_task_edicola/Models/Rivista.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DB_lez06_task_edicola.Models
{
    internal class Rivista : Oggetto
    {
        public string Titolo { get; set; } = null!;
        public string CasaEd { get; set; } = null!;

        public override string ToString()
        {
            return $"[RIVISTA] {Codice} {Prezzo} {Titolo} {CasaEd}";
        }

    }
}

File: Lez_TS/lez09_aggregazione/Rivista.ts
export default class Rivista{
    private codice: string | undefined;
    private titolo: string | undefined;

    constructor(codice?: string, titolo?: string){
        this.codice = codice;
        this.titolo = titolo;
    }

    stampaDettaglio() : void {
        console.log(`Rivista: ${this.codice} ${this.titolo}`)
    }
}